gradle-jdks-setup/src/integrationTest/java/com/palantir/gradle/jdks/setup/GradleJdkInstallationSetupIntegrationTest.java

```diff
@@ -153,7 +159,8 @@ private static void writeFileContent(Path path, String content) throws IOExcepti
         Files.writeString(path, content + "\n");
     }
 
-    private void dockerBuildAndRunTestingScript(String baseImage, String shell, boolean installCurl)
+    private void dockerBuildAndRunTestingScript(
+            String baseImage, String shell, boolean installCurl, boolean addEmptyGradleJdkDir)
```

Comment:
I assumed this meant the `~/.gradle/jdks` dir - maybe call this just `addJdkDir`?